mq/esworker_sns_sqs.py

```diff
@@ -162,7 +164,6 @@ def save_event(self, event, metadata):
                 self.esConnection.save_event(
                     index=metadata['index'],
                     doc_id=metadata['id'],
-                    doc_type=metadata['doc_type'],
```

Comment:
I think we'll need to do some additional stuff here, because previously, mq plugins could and did change the doc_type  of a message, so removing this line means they no longer have that ability (which I think is fine).

 I think we need to modify the mq plugins to instead modify the `type` key on the actual message.

Reply:
I should have added ^^ comment as part of the view, so that it did not post yet haha. I'm scrolling down and am now seeing the mq plugin changes :)

Reply:
so this can be resolved yes?